glib/src/object.rs

```diff
@@ -2554,7 +2554,7 @@ pub unsafe trait ParentClassIs: ObjectType {
 }
 
 /// Automatically implemented by `ObjectSubclass` variants of
-/// [`wrapper!`][crate::glib_wrapper]
+/// [`wrapper!`][crate::wrapper]
```

Comment:
Hmm how should I handle such complaint? maybe I should use `crate::macro::wrapper`?

Reply:
Add a `!` at the end. :)